src/commercial.consentless.ts

```diff
@@ -11,24 +7,32 @@ import { initConsentless } from './lib/consentless/prepare-ootag';
 import { init as setAdTestCookie } from './lib/set-adtest-cookie';
 import { init as setAdTestInLabelsCookie } from './lib/set-adtest-in-labels-cookie';
 
-const bootConsentless = async (consentState: ConsentState): Promise<void> => {
-	/*  In the consented ad stack, we set the ad free cookie for users who
-		don't consent to targeted ads in order to hide empty ads slots.
-		We remove the cookie here so that we can show Opt Out ads.
-		TODO: Stop setting ad free cookie for users who opt out when
-		consentless ads are rolled out to all users.
- 	*/
-	maybeUnsetAdFreeCookie(AdFreeCookieReasons.ConsentOptOut);
-
-	await Promise.all([
+const bootConsentless = async (
+	consentState: ConsentState,
+	isDotcomRendering: boolean,
+): Promise<void> => {
+	const consentlessModuleList = [
 		setAdTestCookie(),
 		setAdTestInLabelsCookie(),
 		initConsentless(consentState),
 		initExclusionSlot(),
 		initFixedSlots(),
 		initArticleInline(),
 		initLiveblogInline(),
-	]);
+	];
+
+	//this is added so that we can load the subscriber cookie for DCR pages and correctly hide ads
```

Comment:
I've added this change as at the moment we don't set the subscriber cookie when a user rejects all on DCR pages. As we've migrated more fronts to DCR this has become a problem, as a user needs to load a page via frontend to user user-features when they've rejected all. Adding this script here fixes the issue for now, but we should probably look to move this outside of the commercial bundle in the future.